code/obj/machinery/door/airlock.dm

```diff
@@ -2005,7 +2005,11 @@ obj/machinery/door/airlock
 			if(!src.arePowerSystemsOn() || (status & NOPOWER))
 				boutput(user, "The door has no power - you can't electrify it.")
 				return
-			if(alert("Are you sure? Electricity might harm a human!",,"No","Yes") == "Yes") // fix for holding spacebar clicking yes
+
+			while (user.client.check_key(KEY_SHOCK))
+				sleep(0.2 SECONDS) // num seems to work fine
```

Comment:
What's this doing?

Reply:
It's because if holding down space when clicking the door, it seems to automatically accept the first tgui prompt option available, which the old way got around by putting "No" as the first option. When testing, the tgui prompt was still doing it for "No."

Reply:
Scream. Ok.